app/.env.example

```
@@ -37,3 +37,11 @@ FINNHUB_API_KEY=<finnhub-api-key>
 # Translation) rather than by HTTP referrer, which a server-side call has none
 # of. First 500k characters a month are free; billing must still be enabled.
 GOOGLE_TRANSLATE_API_KEY=<google-cloud-api-key>
+
+# REQUIRED for the financial statements on the stock page's Reports tab
+# (/api/financials, SEC EDGAR). The SEC's fair-access policy requires every
+# automated request to declare who is making it, as a User-Agent of the form
+# "<app or company> <contact email>", and blocks requests that do not. It is
+# sent only to sec.gov. Without it the route answers "not configured" rather
+# than making a request that would be refused.
+SEC_USER_AGENT=<app name> <contact email>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)